streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/runner/EnvInitializer.java

```diff
@@ -150,49 +130,22 @@ public synchronized void storageInitialize(StorageType storageType) {
       return;
     }
 
-    final String mkdirLog = "storage initialize, now mkdir [{}] starting ...";
-
     FsOperator fsOperator = FsOperator.of(storageType);
     Workspace workspace = Workspace.of(storageType);
 
     // 1. prepare workspace dir
+    String[] dirs =
+        new String[] {
+          workspace.APP_UPLOADS(),
+          workspace.APP_WORKSPACE(),
+          workspace.APP_BACKUPS(),
+          workspace.APP_SAVEPOINTS(),
+          workspace.APP_JARS()
+        };
```

Comment:
How about changed like this?
List<String> dirs = Arrays.as(workspace.APP_UPLOADS(),
          workspace.APP_WORKSPACE(),
          workspace.APP_BACKUPS(),
          workspace.APP_SAVEPOINTS(),
          workspace.APP_JARS()
)